src/main/java/pl/tomaszdziurko/jvm_bloggers/UTMParametrsConstants.java

```diff
@@ -0,0 +1,25 @@
+package pl.tomaszdziurko.jvm_bloggers;
+
+/**
+ * UTM (Urchin Traffic Monitor) URL query parameter identifiers.
+ * 
+ * @author cslysy <jakub.sprega@gmail.com>
+ */
+public class UTMParametrsConstants {
```

Comment:
At first I had the same thoughts but later on I decided to move that constants to separate class. Please noticed that values of BlogPostForMailItem.UTM_SOURCE/UTM_MEDIUM are strictly related to our domain whereas constants from UTMParametrsConstants only define analytics tools conventions keys. That's my view on this point. Do you agree or should I move these constants to BlogPostForMailItem?